### PR TITLE
Setup executor service

### PR DESCRIPTION
**Pull request by Manoharan-Ajay-Anand**

### CODE REVIEW

File: deployment/nginx/sites-enabled/peerprep.sivarn.com [hunk 1]
```diff
@@ -59,6 +59,11 @@ server {
     proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
     proxy_set_header X-Forwarded-Proto $scheme;
   }
+
+  location /api/v1/execute/ {
+    include /etc/nginx/fastcgi_params;
+    fastcgi_pass peerprep-executor-service:9000;
```
Comment (sivayogasubramanian) [inline deployment/nginx/sites-enabled/peerprep.sivarn.com:65]:
```suggestion
    fastcgi_pass peerprep-executor-service:9000/api/v1/execute/;
```

File: deployment/docker-compose.yml [hunk 2]
```diff
@@ -14,6 +14,17 @@ services:
     expose:
       - "27017"
 
+  executor:
```
Comment (sivayogasubramanian) [inline deployment/docker-compose.yml:17]: Add `executor` dependency to `frontend`

**Review by sivayogasubramanian — APPROVED** [review]
LGTM

File: deployment/nginx/sites-enabled/peerprep.sivarn.com (hunk 1 above)
Comment (Manoharan-Ajay-Anand) [reply deployment/nginx/sites-enabled/peerprep.sivarn.com:65]: fastcgi_pass doesn't work like that. can only pass host and port

Comment (sivayogasubramanian) [reply deployment/nginx/sites-enabled/peerprep.sivarn.com:65]: ic okay @Manoharan-Ajay-Anand

File: deployment/docker-compose.yml (hunk 2 above)
Comment (Manoharan-Ajay-Anand) [reply deployment/docker-compose.yml:17]: done

**Pull request merged**